contrib/examples/apiserver/binding.yaml

```diff
@@ -3,4 +3,9 @@ kind: Binding
 metadata:
   name: test-binding
 spec:
+  instanceRef:
+    name: test-instance
   secretName: my-secret
+  configMapName: my-configmap
+  serviceName: my-service
+  osbGuid: b58d64d2-8e01-42b5-a287-bc9ee1860f9d
```

Comment:
I'm inclined to leave the OSBGUIDs out of these examples. I don't think its a good idea to ask end users to have to generate these - let's do it for them.